PWGLF/Tasks/NucleiSpectraTask.cxx

```diff
@@ -139,6 +160,10 @@ DECLARE_SOA_COLUMN(Flags, flags, uint16_t);
 DECLARE_SOA_COLUMN(TPCnsigma, tpcnsigma, uint8_t);
 DECLARE_SOA_COLUMN(TOFnsigma, tofnsigma, uint8_t);
 DECLARE_SOA_COLUMN(TOFmass, tofmass, uint8_t);
+DECLARE_SOA_COLUMN(gPt, genPt, float);
```

Comment:
Hi @mpuccio, same here, could you please move this to the datamodel?

Reply:
the motivation did not change wrt to last time

Reply:
And my comments are still the ones from last time. You said you want to avoid having many different tables, but this is not a general skimming format. These tables are to be used for this task only. If we put it in the data model folder, there is the implicit assumption that anyone can rely on that. This will significantly reduce our flexibility with no additional benefit.

Reply:
I see, let's discuss there then

Reply:
https://github.com/AliceO2Group/O2Physics/pull/2269#discussion_r1147431489